app/connectors/deskpro/domain/DeskProDomain.scala

```diff
@@ -212,3 +212,8 @@ case object AccessibilityProblemTicketConstants extends TicketConstants {
   val auditType: String = "AccessibilityProblemFormSubmission"
   val subject: String   = "Accessibility Problem"
 }
+
+case object ReportOneLoginProblemTicketConstants extends TicketConstants {
+  val auditType: String = "ReportOneLoginProblemFormSubmission"
+  val subject: String   = "One Login for Government Problem Report Submission"
+}
```

Comment:
should we rename stuff to match the url from DTST? so ReportOneLoginComplaint

Reply:
@oscarduignan Agreed, pushed up change for this.